packages/dialtone-vue2/components/rich_text_editor/rich_text_editor.vue

```diff
@@ -600,6 +619,14 @@ export default {
         extensions.push(ConfigurableImage);
       }
 
+      if (this.allowFontColor) {
+        extensions.push(Color);
+      }
+
+      if (this.allowFontFamily) {
+        extensions.push(FontFamily);
+      }
```

Comment:
If they are using a custom font it's likely to fall back to some other font for users that don't have it. Is this okay?

Reply:
i wonder if we can set a default font. but ideally thats what we want if font isnt recognized, go to default

Reply:
nvm it takes the browser default one.